_data/students.yml

```diff
@@ -646,4 +652,4 @@
   facebook: andrewjunggg
   bio: 16 years old. Bandung, Indonesia.
   lat: -6.904950
-  lng: 107.612179
```

Comment:
Can you state why have updated other students info? I hope it's a typo so please revert this change.